src/types/terminal.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+type StartTerminalUri struct {
+	ClusterName   string `uri:"clusterName" binding:"required"`
+	NamespaceName string `uri:"namespaceName" binding:"required"`
+	PodName       string `uri:"podName" binding:"required"`
+	ContainerName string `uri:"containerName" binding:"required"`
+}
+
+type StartTerminalBody struct {
+	Shell string `json:"shell" binding:"required,oneof=bash sh powershell cmd"`
+}
+
+// StartTerminalResponse is sent by HandleStartTerminal. The ID is a random session id that binds the original REST request and the SockJS connection.
+// Any client api in possession of this ID can hijack the terminal_utils session.
```

Comment:
For my understanding, If I open a terminal session using postman (for example) and get a session ID, Can I then send another request in another tab using that ID and get into the same session? what happens to the original tab? Is it still active?